bin/install-wp-tests.sh

```diff
@@ -1,151 +1,76 @@
-			else
-				local ARCHIVE_NAME="wordpress-$LATEST_VERSION"
-			fi
-		else
-			local ARCHIVE_NAME="wordpress-$WP_VERSION"
-		fi
-		download https://wordpress.org/${ARCHIVE_NAME}.tar.gz  $TMPDIR/wordpress.tar.gz
-		tar --strip-components=1 -zxmf $TMPDIR/wordpress.tar.gz -C $WP_CORE_DIR
-	fi
-
-	download https://raw.github.com/markoheijnen/wp-mysqli/master/db.php $WP_CORE_DIR/wp-content/db.php
-}
-
-install_test_suite() {
-	# portable in-place argument for both GNU sed and Mac OSX sed
-	if [[ $(uname -s) == 'Darwin' ]]; then
-		local ioption='-i .bak'
-	else
-		local ioption='-i'
-	fi
-
-	# set up testing suite if it doesn't yet exist
-	if [ ! -d $WP_TESTS_DIR ]; then
-		# set up testing suite
-		mkdir -p $WP_TESTS_DIR
-		svn co --quiet --ignore-externals https://develop.svn.wordpress.org/${WP_TESTS_TAG}/tests/phpunit/includes/ $WP_TESTS_DIR/includes
-		svn co --quiet --ignore-externals https://develop.svn.wordpress.org/${WP_TESTS_TAG}/tests/phpunit/data/ $WP_TESTS_DIR/data
+# shellcheck disable=SC1091
+source "$(dirname "$0")/helpers.sh"
+
+# Initialize variables with default values
+TMPDIR="/tmp"
+DB_NAME="wordpress_test"
+DB_USER="root"
+DB_PASS=""
+DB_HOST="127.0.0.1"
+WP_VERSION=${WP_VERSION:-latest}
+SKIP_DB=""
+
+# Parse command-line arguments
+for i in "$@"; do
+	# Skip 'bash' argument
+	if [[ $i == "bash" ]]; then
+		echo "Ignoring 'bash' argument"
+		continue
 	fi
 
-	if [ ! -f wp-tests-config.php ]; then
-		download https://develop.svn.wordpress.org/${WP_TESTS_TAG}/wp-tests-config-sample.php "$WP_TESTS_DIR"/wp-tests-config.php
-		# remove all forward slashes in the end
-		WP_CORE_DIR=$(echo $WP_CORE_DIR | sed "s:/\+$::")
-		sed $ioption "s:dirname( __FILE__ ) . '/src/':'$WP_CORE_DIR/':" "$WP_TESTS_DIR"/wp-tests-config.php
-		sed $ioption "s/youremptytestdbnamehere/$DB_NAME/" "$WP_TESTS_DIR"/wp-tests-config.php
-		sed $ioption "s/yourusernamehere/$DB_USER/" "$WP_TESTS_DIR"/wp-tests-config.php
-		sed $ioption "s/yourpasswordhere/$DB_PASS/" "$WP_TESTS_DIR"/wp-tests-config.php
-		sed $ioption "s|localhost|${DB_HOST}|" "$WP_TESTS_DIR"/wp-tests-config.php
+	# Skip the script path argument
+	if [[ $i == *install-wp-tests.sh ]]; then
+		echo "Ignoring script path argument"
+		continue
 	fi
 
-}
-
-install_db() {
+	case $i in
+		--dbname=*)
+		DB_NAME="${i#*=}"
+		;;
+		--dbuser=*)
+		DB_USER="${i#*=}"
+		;;
+		--dbpass=*)
+		DB_PASS="${i#*=}"
+		;;
+		--dbhost=*)
+		DB_HOST="${i#*=}"
+		;;
+		--version=*)
+		WP_VERSION="${i#*=}"
+		;;
+		--skip-db=*)
+		SKIP_DB="true"
+		;;
+		--tmpdir=*)
+		TMPDIR="${i#*=}"
+		;;
+		*)
+		# unknown option
+		echo "Unknown option: $i. Usage: ./bin/install-wp-tests.sh --dbname=wordpress_test --dbuser=root --dbpass=root --dbhost=localhost --version=latest --tmpdir=/tmp --skip-db=true"
+		exit 1
+		;;
+	esac
+done
 
-	if [ ${SKIP_DB_CREATE} = "true" ]; then
-		return 0
-	fi
+WP_TESTS_DIR=${WP_TESTS_DIR-$TMPDIR/wordpress-tests-lib}
+WP_CORE_DIR=${WP_CORE_DIR-$TMPDIR/wordpress/}
```

Comment:
```suggestion
WP_TESTS_DIR=${WP_TESTS_DIR:-$TMPDIR/wordpress-tests-lib}
WP_CORE_DIR=${WP_CORE_DIR:-$TMPDIR/wordpress/}
```
Not sure if bash plays nice with variable in the substitution here.